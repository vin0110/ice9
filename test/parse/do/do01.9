do 1 -> a; od
