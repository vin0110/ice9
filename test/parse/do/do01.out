DO:
  ILIT:1
(body)
  EXP:
    ID:'a'
